app/main.py

```diff
@@ -1,2 +1,53 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.resize = 2
+        self.length = 0
+        self.hash = [[] for _ in range(self.capacity)]
+
+    def __len__(self) -> int:
+        return self.length
+
+    def get_index_and_hash(self, key: Any) -> Any:
```

Comment:
```suggestion
    def get_index_and_hash(self, key: Hashable) -> Any:
```
Please fix all over the project